core/src/com/unciv/ui/overviewscreen/GlobalPoliticsOverviewTable.kt

```diff
@@ -168,16 +169,21 @@ class GlobalPoliticsOverviewTable (
         return wonderTable
     }
 
+    private fun getCivName(otherciv: CivilizationInfo): String {
+        if (viewingPlayer.knows(otherciv)) {
+            return otherciv.civName
+        }
+        return "an unknown civilization"
+    }
+
     private fun getPoliticsOfCivTable(civ: CivilizationInfo): Table {
         val politicsTable = Table(skin)
 
         // wars
         for (otherCiv in civ.getKnownCivs()) {
             if(civ.isAtWarWith(otherCiv)) {
-                val warText = if (viewingPlayer.knows(otherCiv))
-                    "At war with [${otherCiv.civName}]".toLabel()
-                else
-                    "At war with an unknown civilization".toLabel()
+                println(getCivName(otherCiv))
```

Comment:
Remove

Reply:
I'll do it

Reply:
Oops I can't believe that slipped away after reading the changes at least 2 times :))